tap_egencia/streams.py

```diff
@@ -30,13 +31,19 @@ class TransactionsStream(egenciaStream):
 
     name = "transactions-api"
     schema = PropertiesList(
+        Property("failure-response", StringType),
         Property("report_id", StringType),
         Property("metadata", metadataObject),
         Property("transactions", transactionsObject),
         Property("_links", linksObject)
         ).to_dict()
     authenticator = None
 
+    def dict_to_binary(the_dict):
```

Comment:
I see this function being added, but I don't see it being called anywhere?

Reply:
Thats left in by mistake. removing. good catch